src/DotNetLightning.Core/Serialize/Features.fs

```diff
@@ -230,6 +224,29 @@ type FeatureBit private (bitArray) =
     override this.ToString() =
         this.BitArray.PrintBits()
         
+    member this.SetFeatureBit(index: int, on: bool): unit =
+        let length = this.BitArray.Length
+        if length <= index then
+            this.BitArray.Length <- index + 1
+
+            //this.BitArray.RightShift(index - length + 1)
+
+            // NOTE: Calling RightShift gives me:
+            // "The field, constructor or member 'RightShift' is not defined."
```

Comment:
Probably because we are targeting netstandard2.0 instead of 2.1.
I forgot why we are still using 2.0.

Defining by ourselves is fine but ideally you can 
1. change `<TargetFramework>netstandard2.0` to `<TargetFrameworks>netstandard2.0;netstandard2.1`
1. use compilation flag to use RightShift only when the target is netstandard2.1

Reply:
I'm not sure that's a good idea. If we use a compilation flag then we'll still need to implement `RightShift` ourselves for when we're using 2.0. So it wouldn't help reduce bugs or make the code smaller or simpler. It would just make it inconsistent between two slightly different build configurations.